app/scripts/background.js

```diff
@@ -786,7 +786,10 @@ browser.runtime.onInstalled.addListener(({ reason }) => {
 });
 
 function setupSentryGetStateGlobal(store) {
-  global.getSentryState = function () {
+  if (!global.rootGlobals) {
+    global.rootGlobals = {};
```

Comment:
This will exist already in production builds because it will be created by `lavamoat-runtime.js`. But in development, it's constructed here.